tests/end_to_end/test_helper.cpp

```diff
@@ -259,8 +263,7 @@ GlueCloseHandle(HANDLE object_handle)
     return TRUE;
 }
 
-bool
-GlueDuplicateHandle(
+_Success_(return != false) bool GlueDuplicateHandle(
```

Comment:
curious why we're using `return != false` here but use `return == true` for `bool GlueDeviceIoControl...` (line 437 below)?

Reply:
Switch all to return != false.